tsconfig.json

```diff
@@ -21,7 +21,8 @@
         "pretty": true,
         "composite": true,
         "importsNotUsedAsValues": "error",
-        "extendedDiagnostics": true
+        "extendedDiagnostics": true,
+        "ignoreDeprecations": "5.0"
```

Comment:
importsNotUsedAsValues is deprecated, need to figure out what we want to do in the future, it still works for now